opencti-platform/opencti-graphql/src/domain/subType.js

```diff
@@ -1,5 +1,41 @@
-import { querySubTypes, querySubType } from '../database/middleware';
+import * as R from 'ramda';
+import { ABSTRACT_STIX_CORE_RELATIONSHIP, ABSTRACT_STIX_DOMAIN_OBJECT, schemaTypes } from '../schema/general';
+import { STIX_SIGHTING_RELATIONSHIP } from '../schema/stixSightingRelationship';
+import { buildPagination } from '../database/utils';
 
 export const findById = (subTypeId) => querySubType(subTypeId);
 
 export const findAll = (args) => querySubTypes(args);
+
+export const querySubType = async (subTypeId) => {
```

Comment:
Not sure to understand this part ?

Reply:
I just moved the querySubTypes method from middleware.js to this file.